mountpoint-s3.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: mountpoint-s3
+  version: 1.10.0
+  epoch: 0
+  description: A simple, high-throughput file client for mounting an Amazon S3 bucket as a local file system.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-19
+      - cmake
+      - fuse3
+      - fuse3-dev
+      - libclang-cpp-19
+      - pkgconf
+      - rust
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/awslabs/mountpoint-s3
+      tag: v${{package.version}}
+      expected-commit: 2b0161600400c7ab7ccb1d6811abb9774facef5d
+      recurse-submodules: true
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/bin
+      cargo build --release
+      install -Dm755 target/release/mount-s3 "${{targets.contextdir}}"/usr/bin/mount-s3
+
+  - uses: strip
+
+test:
+  pipeline:
+    - runs: |
+        mount-s3 --version
+
+update:
+  enabled: true
+  github:
+    identifier: awslabs/mountpoint-s3
+    strip-prefix: mountpoint-s3-
```

Comment:
the releases contains a space [mountpoint-s3 v1.10.0](https://github.com/awslabs/mountpoint-s3/releases/tag/mountpoint-s3-1.10.0) in there wondering if we will be able to get update on this here if we use `-` 

looking at API output from GitHub, If we are filtering based on tag_name then this looks good but if based on name then maybe we can use tags. 

`gh api /repos/awslabs/mountpoint-s3/releases` 
```json
    "node_id": "RE_kwDOH3Saf84KuwPA",
    "tag_name": "mountpoint-s3-1.10.0",
    "target_commitish": "main",
    "name": "mountpoint-s3 v1.10.0",
```